lumicks/pylake/tests/test_imaging_confocal/test_kymo.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+import pytest
+from lumicks.pylake.channel import Continuous, Slice
+
+from ..data.mock_confocal import (
+    axes_dict_list,
+    confocal_mock_file_h5,
+    generate_kymo_with_ref,
+    generate_scan_json,
+)
+
+
+# CAVE: If you want to test a cached property, after having modified parameters that change the
+# value of the property, ensure to clear the `_cache` attribute before and after testing. To achieve
+# both, you can monkeypatch the `_cache` attribute with an empty dict.
+@pytest.fixture(scope="module")
+def test_kymos():
+    return {"standard": generate_kymo_with_ref("standard", np.random.poisson(10, (5, 4, 3)))}
+
+
+@pytest.fixture
+def add_force_channel(monkeypatch):
+    def do_add_force_channel(kymo, ref, head=30, tail=10, head_lines=2, channel="force1x"):
+
+        # Force channel that overlaps kymo; step from high to low force
+        # Per default two lines of the kymo have a force of 30 and the rest of the lines 10.
+        samples_per_line = (
+            ref.metadata.pixels_per_line * ref.infowave.samples_per_pixel
+            + 2 * ref.infowave.line_padding
+        )
+        number_of_samples = ref.metadata.lines_per_frame * samples_per_line
+        # First line starts after line_padding
+        first_pixels = ref.infowave.line_padding + head_lines * samples_per_line
+        force = np.r_[
+            np.ones(first_pixels) * head, np.ones(number_of_samples - first_pixels) * tail
+        ]
+
+        monkeypatch.setattr(
+            kymo.file,
+            channel,
+            Slice(Continuous(force, ref.start, ref.timestamps.dt)),
+            raising=False,
+        )
```

Comment:
This relies on that you access it over the attribute rather than through `__getitem__`. I guess that if you'd need more, you would sooner go through the full file creation with the `Mock` classes?